src/descriptors/power_system_structs.json

```diff
@@ -2000,6 +2000,322 @@
       ],
       "supertype": "DynamicInjection"
     },
+    {
+      "struct_name": "GenericDynamicLoad",
```

Comment:
Why Generic? is there something more descriptive?

Reply:
Is because of the paper. Maybe we can go with `ExponentialDynamicLoad`?

Reply:
Yeah, that is better